vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/info/ProjectLinksComponent.java

```diff
@@ -50,11 +56,23 @@ public ProjectLinksComponent(@Autowired ProjectLinkingService projectLinkingServ
     Objects.requireNonNull(projectLinkingService);
     initOfferSearch(offerLookupService);
     initProjectLinksGrid();
-    getContent().addTitle(TITLE);
-    getContent().addContent(offerSearch, projectLinks);
+
+    initLayout();
+
     projectLinksComponentHandler = new ProjectLinksComponentHandler(projectLinkingService);
   }
 
+  private void initLayout() {
+    addClassName("attachments-area");
+    Span titleSpan = new Span();
+    titleSpan.add(TITLE);
```

Comment:
should be `setText` I think

Reply:
This is directly taken from the way we did it before. Did you test if setText works as well?